components/Login.js

```diff
@@ -55,6 +55,8 @@ export default function Login({ children }) {
         <Header>Personal Shopping Made Easy</Header>
         <h4>Create Your Personal Shopping List And Keep Track </h4>
         <button onClick={() => signIn()}>Sign in</button>
+        <br />
+        <button onClick={() => signIn()}>Sign in with Google</button>
```

Comment:
Both buttons will lead to the same page, since they make use of `signIn`.

I would remove the second button.

```suggestion
        <button onClick={() => signIn()}>Sign in</button>
```

Reply:
Hi Niklas,

We have implemented 2 sign ins - Sign in with Github and Sign in with Google. I forgot to rename the first button from Sign in  to Sign in with GitHub and hence the confusion. Is this okay?

Reply:
Ah, I see.

The buttons still are a bit confusing to me:
When I click on "Login with GitHub" I get redirected to a page where I can log in with Google or Credentials.
When I click on "Login with Google" I can log in directly with Google.

But this might also change, once both login branches are in main and everything is aligned, I assume.

Reply:
Yes Niklas. Actually, I was wrong. I realised this when Felix explained. So now we have a single button called Sign in. Felix just helped us with this. Thanks again